Add TIGEM config file (#422)
nf-core/configs: docs/tigem.md

# nf-core/configs: TIGEM configuration

To use, run the pipeline with `-profile tigem`. This will download and launch the tigem.config which has been pre-configured with a setup suitable for the TIGEM personal biocluster.

---

> For this pipeline you need a docker container containing the tools needed for the specific pipeline. You can find the docker containers on giusmar docker hub profile. A docker image containing the required software will be downloaded, and converted to a Singularity image before execution of the pipeline.

Reviewer: Maybe reformulate this a bit - it reads as if the user has to do anything (which they really don't apart from waiting) ;-)

Author: I want to use a different docker container for each different pipeline, but it is for my configuration.Exactly, nf-core pipelines set a different docker container for each process automatically, so, basically, the better reformulation is to delete the sentence :)
I ll update the doc with further details for 
`process.executor = 'slurm'
google.zone = 'europe-west1'`

---

> This pipeline can be used on TIGEM clusters, in which is installed slurm as job scheduling system, so you can use this config if you have the same tool installed. An additional parameter is google.zone to allow downloading data from GCP for a specific time zone. It should not interfere with a local or an AWS configuration.

Reviewer: ```suggestion
This configuration profile can be used on TIGEM clusters, with the pre-installed SLURM job scheduling system. An additional parameter is `google.zone` to allow downloading data from GCP for a specific time zone. It should not interfere with any local or other AWS configuration.
```

Author: Thank you so much

Reviewer: No this should be fine now.
